great-idea-website/index.html

```diff
@@ -15,49 +15,89 @@
 </head>
 
   <body>
+    <div class="main-container">
+   
+      <nav>
+          <a href="services">Services</a>
+          <a href="product">Product</a>
+          <a href="vision">Vision</a>
+          <a href="features">Features</a>
+          <a href="about">About</a>
+          <a href="contact">Contact</a>
```

Comment:
your hrefs have to link to another html document in your project.  I'm not sure if these links will actually go to other pages.  need to check and see if you have to add the .html at the end.  and if the files are ever in another directory, you have to specify the directory path : directory/services.html for example.